<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeMapSettingsDialog</class>
 <widget class="QWidget" name="ChangeMapSettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Map Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLineEdit" name="line_panorama">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radio_panorama">
       <property name="text">
        <string>Panorama Graphic</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radio_fog">
       <property name="text">
        <string>Fog Graphic</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="line_fog">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="line_battleback">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="radio_battleback">
       <property name="text">
        <string>Battleback Graphic</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="button_panorama">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="button_fog">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QToolButton" name="button_battleback">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radio_panorama</tabstop>
  <tabstop>line_panorama</tabstop>
  <tabstop>button_panorama</tabstop>
  <tabstop>radio_fog</tabstop>
  <tabstop>line_fog</tabstop>
  <tabstop>button_fog</tabstop>
  <tabstop>radio_battleback</tabstop>
  <tabstop>line_battleback</tabstop>
  <tabstop>button_battleback</tabstop>
  <tabstop>button_cancel</tabstop>
  <tabstop>button_ok</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radio_panorama</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_panorama</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_panorama</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_panorama</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_fog</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_fog</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_fog</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_fog</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_battleback</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_battleback</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_battleback</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_battleback</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
